packages/authorization-server/src/model/domain/models.ts

```diff
@@ -1,5 +1,24 @@
+import { IncomingHttpHeaders } from "http";
+import { authorizationServerApi } from "pagopa-interop-api-clients";
+import { z } from "zod";
+
 export interface InteropTokenResponse {
   access_token: string;
   token_type: string;
   expires_in: number;
 }
+
+export type TokenRequest = {
+  headers: IncomingHttpHeaders & { DPoP?: string };
+  body: authorizationServerApi.AccessTokenRequest;
+};
```

Comment:
Is this used anywhere? I can't find any implementation 
(searched for it in the branch of the [other PR](https://github.com/pagopa/interop-be-monorepo/pull/1920/files))